static/oas/v1/sports/components/logos.yml

```diff
@@ -35,17 +36,18 @@ paths:
           required: true
           description: The ID of the logo to retrieve
           schema:
-            type: string
+            type: integer
       responses:
         200:
-          description: The response body should contain the logo binary
+          description: The response body should contain the png logo
           content:
             multipart/form-data:
```

Comment:
Update this to image/png, since that is the content type.

Reply:
Done